.github/workflows/gh-ci-cron.yaml

```diff
@@ -196,3 +196,71 @@ jobs:
     - name: run_tests
       run: |
         pytest -n $numprocs testsuite/MDAnalysisTests --disable-pytest-warnings --durations=50
+        
+  # Issue 4208
+  conda-latest-release:
+    # A set of runner to check that the latest conda release works as expected
+    if: "github.repository == 'MDAnalysis/mdanalysis'"
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+    strategy:
+        fail-fast: false
+        matrix:
+          os: [ubuntu-latest, macos-latest]
```

Comment:
Why do we not run the conda based installation on windows? (Just out of curiosity.)

Reply:
Because the conda check if a "full dependency" check it installs things that aren't windows compatible (TNG, etc...). 

My thought here is that we could have a Windows runner that just doesn't do a full dependency install, but that essentially amounts to the pip check? Maybe I'm missing something important though.

Reply:
Thanks for the explanation. Does not seem to be necessary to do more right now — the pip run is functionally equivalent.